lua/entities/wired_door/init.lua

```diff
@@ -5,6 +5,20 @@ include('shared.lua')
 
 ENT.WireDebugName = "Door Controller"
 
+local function switch(t)

```

Comment:
moved from being a global at the bottom (i think this is stupid and shouldnt exist but it definitely shouldnt be a global)